### DISCUSSION TITLE
Subtitles not shown

### DISCUSSION THREAD

**Discussion by pherrmann871** [aclap-dev/video-downloadhelper#1291] [Help and Q&A]
### Your question

Hi,
After downloading a video with subtitles, all is OK excepted missing all subtitles (in french).
OS : Linux antiX 19 (64)
Firefox 115.12.0esr (64)
VDH 9.1.0.44
VDHcoapp 2.0.19
Link to this video : [https://www.arte.tv/fr/videos/114211-000-A/birds-of-america/](url)
Did I something wrong ?
Thank you

### Addon & CoApp version and details

~/.local/share/vdhcoapp/vdhcoapp --info
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "/home/herrmann/.local/share/vdhcoapp/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "linux",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "/home/herrmann",
  "converterBinary": "/home/herrmann/.local/share/vdhcoapp/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360"
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

For some reason, the link you provided does not just go to that web page when you click it here on GitHub.  It goes to a 404 page not found page here on GitHub.  I had to manually copy/paste the string into another Firefox window to visit the page.

Unfortunately, I'm getting the pas disponible dans votre pays message.  Dommage . . .

However, you should read #956.  Others have encountered similar issues.  Captions are not supported by VDH.  They never have been.  Supposedly, @paulrouget Paul is going to add that support to the product some day.  But we don't have even an estimate of when that will happen.  For the time being, you have to download the subtitles yourself, separately, using ffmpeg or Firefox.  You have to understand that the captions are not actually part of the video.  They are in a separate file that you need to download separately.  In #956 I show how to download the subtitles for a video on arte.tv.  You would need to use a similar technique.

**Reply by pherrmann871** [reply]
GrandpaWildWilly gave me a solution that works great !
Thanks for all
